html_versions.py

```diff
@@ -0,0 +1,239 @@
+"""
+Inspired by Tom's /dls/cs-www/reports/scripts/list_ioc_support_modules.py
+"""
+
+import pkg_resources
+pkg_resources.require('jinja2')
+import jinja2
+
+from convert import configuration
+from convert import coordinates
+from convert import dependency
+from convert import launcher
+from convert import utils
+
+import os
+import subprocess
+import ConfigParser
+import logging as log
+LOG_FORMAT = '%(levelname)s:  %(message)s'
+LOG_LEVEL = log.WARNING
+log.basicConfig(format=LOG_FORMAT, level=LOG_LEVEL)
+
+SUPPORT = '/dls_sw/prod/R3.14.12.3/support'
+IOC = '/dls_sw/prod/R3.14.12.3/ioc'
+RESOURCES_DIR = 'res'
```

Comment:
You use "/dls_sw/prod/R3.14.12.3" again later. Might be cleaner to define BASE_PATH and the SUPPORT and IOC areas as strings